esyi/Package.ml

```diff
@@ -580,7 +580,7 @@ end
 module File = struct
   [@@@ocaml.warning "-32"]
   type t = {
-    name : Path.t;
+    name : string;
```

Comment:
FYI, I converted this to `string` so that we could preconvert the Windows path separators via `\\` -> `/`. If we leave this as a Path, it seems to use the default string coercion strategy, which uses platform-specific separators. Let me know if you have a better idea for resolving this, though!

Reply:
Let's make `File.t` abstract then? `string` is a bit unsafe because it's easy to use it as a path segment `Path.(base / file.name)` which throws at runtime.